mfterm.c

```diff
@@ -316,7 +316,7 @@ char* completion_spec_generator(const char* text, int state) {
       continue;
 
     char* fname = inst->field->name;
-    size_t fname_len = strlen(fname);
+    size_t fname_len = strlen(fname)+1;
 
     // Check if the field is applicable - right prefix
     if (fname_len >= parent_end_len &&
```

Comment:
```

```suggestion
     if (fname_len >= parent_end_len &&😞
```

```